addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -168,6 +170,49 @@ const PopupWidget = publicWidget.Widget.extend({
             return;
         }
         this.$target.find('.modal').modal('show');
+        this.releaseFocus = this._trapFocus();
+    },
+    /**
+     * Traps the focus within the modal.
+     * Returns a function that refocuses the element that was focused before the
+     * modal opened.
+     *
+     * @returns {Function}
+     */
+    _trapFocus() {
+        const tabableEls = getTabableElements(this.el);
```

Comment:
Here, you're assuming that the tabable elements won't change while the popup is open. Are we sure about that? If not, we can recompute the tabable elements in the _onKeydown event?